ibis/sql/alchemy.py

```diff
@@ -100,6 +100,25 @@ def sa_boolean(_, satype, nullable=True):
     return dt.Boolean(nullable=nullable)
 
 
+@dt.dtype.register(MySQLDialect, sa.types.Numeric)
```

Comment:
it seems the tests have some conflict between float and decimal
maybe you can try `sqlalchemy.types.DECIMAL`

Reply:
Really, I'm not particularly interested in MySQL right now (just Postgres), but I'm having a tough time deciding how to handle when the generic SQLAlchemy backend should be used, and when a more specific backend should be used. In particular, I'm not really sure how the multiple dispatch handles this case.

Reply:
```
    -    float_col : float32
    +    float_col : decimal(1000, 0)
```
it seems the test for python 3.5 has this problem between float and decimal
maybe because the libraries  in this environment is no updated.

Reply:
Thanks for taking a look. I'm not sure what you mean here, what library is not updated?